File: sqlUpdate.py
import MySQLdb

#Open database connection
db =  MySQLdb.connect (host="localhost",user="user",passwd="password",db="school")

#prep a cursor object using cursor() method; cursor is like a navigation method
db.autocommit(True)
cursor = db.cursor(MySQLdb.cursors.DictCursor)

name='derek'
age=12
gradeLevel=10

sql = "UPDATE students SET age=16 WHERE name='derek'"
cursor.execute(sql)

cursor.close()
db.close()

File: sqlUpdate.php
<?php
$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";

$conn = new mysqli($servername, $username, $password, $dbname);

// check connection
if($conn->connect_error){
    die("connection failed:".$conn->connect_error);
}

$sql = "UPDATE students SET name='derek' WHERE name='john'";
$result =$conn->query($sql);

if ($conn->query($sql) === TRUE) {
    echo "Record updated successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>
